src/pages/Main/components/Product/Product.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import './Product.scss'
+
+function Product() {
+  return (
+    <div className="product">
+      <div className="productList">
+        {PRODUCT_DATA.map((item, index) => {
+          return (
+            <div key={index} className="productItem">
```

Comment:
```suggestion
        {PRODUCT_DATA.map((item) => {
          return (
            <div key={item.id} className="productItem">
```

key prop에는 고유한 값이 들어가면 되는데, 데이터에 고유한 값이 있다면
index 값을 사용하지 않아도 됩니다!